http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -106,8 +111,11 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       _ <- logger.info(ctx.loggingCtx)(
         s"Finished app creation for app ${params.appName.value} in cluster ${landingZoneResources.clusterName.value} in cloud context ${params.cloudContext.asString}"
       )
+      // Poll app status
+      _ <- streamFUntilDone(
+          cromwellDao.getStatus(CloudContext.Azure(params.cloudContext), RuntimeName(params.appName.value)),
+        maxAttempts=6, delay=10 seconds).interruptAfter(1 minute).compile.lastOrError
```

Comment:
I think want to increase this to something like 20 minutes total, polling every 10 seconds. We actually define this in config [here](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/resources/reference.conf#L793-L797), and read it for GKE [here](https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala#L1173-L1175). We can probably reuse the config objects for AKS, I think.

And as mentioned above, let's get rid of the `RetryPolicy` in `HttpCromwellDAO`, so all polling happens here.

Reply:
ok sounds good yeah this was just a temporary measure.